components/compliance-service/reporting/relaxting/assets.go

```diff
@@ -441,3 +441,41 @@ func (backend ES2Backend) getUnCollectedAssets(ctx context.Context, from int32,
 	filtQuery.Must(getStartTimeAndEndTimeRangeForAsset(filters))
 	return backend.getAssetsList(ctx, from, size, filtQuery)
 }
+func (backend ES2Backend)GetAsset(ctx context.Context, filters map[string][]string , size int32 , from int32 , assetType string) ([]*reportingapi.Assets, error) {
+    boolquery := backend.getFiltersQueryForAssetFilters(filters)
+    assets := make([]*reportingapi.Assets, 0)
+    if assetType == "unreachable" {
```

Comment:
first condition should we of null check. And have a enum in constants file for all the type of asests